errata/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <!--
+    The data-githubrepo attribute provides the owner/repo name on github. For W3C repositories most of those are of the
+    form 'w3c/XXX', although there are groups that have their own owner for their repository.
+  -->
+  <head data-githubrepo="w3c/rdf-canon">
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    <title>Open Errata for the RDF-CANON specification published by the RCH Working Group</title>
```

Comment:
```suggestion
    <title>Open Errata for the “RDF Dataset Canonicalization and Hash” (RDF-CANON) specification published by the RCH Working Group</title>
```

Reply:
Wouldn't it be better to omit 'and Hash' here?
```suggestion
    <title>Open Errata for the “RDF Dataset Canonicalization” (RDF-CANON) specification published by the RCH Working Group</title>
```

Reply:
> Wouldn't it be better to omit 'and Hash' here?

Yes. Updated my suggestion.